core/src/main/java/org/eclipse/hono/tracing/TracingHelper.java

```diff
@@ -181,7 +181,24 @@ public static void setDeviceTags(final Span span, final String tenantId, final S
      * @throws NullPointerException if error is {@code null}.
      */
     public static void logError(final Span span, final Throwable error) {
-        logUnexpectedError(error, span);
+        logError(span, error, false);
+    }
+
+    /**
+     * Marks an <em>OpenTracing</em> span as erroneous and logs an exception.
+     * <p>
+     * This method does <em>not</em> finish the span.
+     *
+     * @param span The span to mark.
+     * @param error The exception that has occurred.
+     * @param skipUnexpectedErrorCheck Whether to skip the check for an unexpected error, meaning no WARN log entry will
+     *            be created for such an error.
```

Comment:
FMPOV the *logError(Span, Throwable)* method's default behavior (of passing in `false`) should be documented in its JavaDoc.